django/db/backends/sqlite3/schema.py

```diff
@@ -11,26 +11,20 @@
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_table = "DROP TABLE %(table)s"
-    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s)"
+    sql_create_fk = None
+    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
```

Comment:
It's probably feasible to create a script or management command or similar to rebuild tables. I wasn't sure if it's needed; i.e. what circumstances will create a problem with non-upgraded tables? The problems in the test suite related to that seemed to be with `flush` which doesn't like a huge concern (if someone wants to do that, just delete the tables instead).